third_party/prepare.sh

```diff
@@ -1,15 +1,16 @@
 #!/bin/bash -e
 
-git_version=$(git --version)
-if [ >= "1.8.4" ]; then
-    echo "Since git 1.8.4 (August 2013), you don't have to be at top-level to run git submodule update."
+cd $(dirname $0)
+
+requiredGitVersion="1.8.4"
+currentGitVersion="$(git --version | awk '{print $3}')"
+if [ "$(printf '%s\n' "$requiredGitVersion" "$currentGitVersion" | sort -V | head -n1)" = "$requiredGitVersion" ]; then
+    echo "Found Git: $(which git). (foud version $currentGitVersion, required version >= $requiredGitVersion)"
```

Comment:
This line is useless, we don't need an extra confirm when we have the correct version installed.

Reply:
I wiil fix it soon.